core/src/consensus/stake/actions.rs

```diff
@@ -26,13 +29,17 @@ use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
 use crate::client::ConsensusClient;
 use crate::consensus::{ConsensusMessage, ValidatorSet};
 
-const ACTION_TAG_TRANSFER_CCS: u8 = 1;
-const ACTION_TAG_DELEGATE_CCS: u8 = 2;
-const ACTION_TAG_REVOKE: u8 = 3;
-const ACTION_TAG_SELF_NOMINATE: u8 = 4;
-const ACTION_TAG_REPORT_DOUBLE_VOTE: u8 = 5;
-const ACTION_TAG_REDELEGATE: u8 = 6;
-const ACTION_TAG_CHANGE_PARAMS: u8 = 0xFF;
+#[derive(FromPrimitive)]
+#[repr(u8)]
+enum ActionTag {
+    TRANSFER_CCS = 1,
```

Comment:
Usually, the Pascal case(camel case with the first capital letter) is used for enum variants.
https://doc.rust-lang.org/1.0.0/style/style/naming/README.html